core/constants-dev.yaml

```diff
@@ -51,7 +51,7 @@ ntpPollDelay: 300000000 # 300 sec
 # Misc
 maxLocalTxs: 10000
 protocolMagic: 0
-enhancedMessageBroadcast: 2
```

Comment:
Both parameters should exist! First is for number of nodes per bucket we take, second is for minimal time node is in bucket